### DISCUSSION TITLE
Wave Server

### DISCUSSION THREAD

**Discussion by Rmfjd1978** [h2oai/wave#1848] [Q&A]
Hi 

I've a wave server mounted on a linux container where the service (waved) after some uses shuts down.
How can i alter the log level so i can understand the problem?

**Comment by mturoci** [comment]
There is no more logging than what you get by default. All errors/problems should be logged though. Can you maybe paste logs here so I could point you in the right direction?

**Comment by Rmfjd1978** [comment]
The server logs goto a specific folder or only stdout?

**Reply by mturoci** [reply]
Stdout, which can be redirected to file if needed.

**Comment by Rmfjd1978** [comment]
Hi 

Here is an Excel file with the logs.
The logs ara in a strange format because they are produce by a logstash and inserted in ElasticSearch.

[WaveServerLogs.xlsx](https://github.com/h2oai/wave/files/10806401/WaveServerLogs.xlsx)

**Reply by mturoci** [reply]
These seem incomplete though. One interesting thing though is the `stream: stderr` part during download. I would start looking for problems during download first.
